src/SSL/Entities/CheckedCertificate.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace UKFast\SDK\SSL\Entities;
+
+class CheckedCertificate
+{
+    /**
+     * @var string
+     */
+    public $validFrom;
+
+    /**
+     * @var string
+     */
+    public $validTo;
+
+    /**
+     * @var string
+     */
+    public $issuer;
+
+    /**
+     * @var string
+     */
+    public $signatureAlgorithm;
+
+    /**
+     * @var string
+     */
+    public $domainCovered;
+
+    /**
+     * @var string
+     */
+    public $ip;
+
+    /**
+     * @var string
+     */
+    public $hostname;
+
+    /**
+     * @var string
+     */
+    public $port;
+
+    /**
+     * @var string
+     */
+    public $currentTime;
+
+    /**
+     * @var string
+     */
+    public $serverTime;
+
+    /**
+     * @var string
+     */
+    public $serverSoftware;
+
+    /**
+     * @var string
+     */
+    public $domainsSecured;
+
+    /**
+     * @var string
+     */
+    public $serialNumber;
+
+    /**
+     * @var string
+     */
+    public $multiDomain;
+
+    /**
+     * @var string
+     */
+    public $wildcard;
+
+    /**
+     * @var string
+     */
+    public $heartbleedVulnerable;
```

Comment:
Could we have a key-pair array of vulnerabilities here? Then we can map the API array into here and we can add new vulnerabilities without updating the SDK

Reply:
moved the vulnerabilities into an array